server/publications/collections/products.js

```diff
@@ -320,14 +320,6 @@ Meteor.publish("Products", function (productScrollLimit = 24, productFilters, so
     return this.ready();
   }
 
-  // Get active shop id's to use for filtering
```

Comment:
with the fix, I was able to move this code down, saving a query when in editMode